static/app/utils/replays/frame.tsx

```diff
@@ -74,11 +73,13 @@ export function getBreadcrumbType(frame: ReplayFrame): BreadcrumbType {
       case 'issue':
         return BreadcrumbType.ERROR;
       case 'ui.slowClickDetected':
-        return (frame as SlowClickFrame).data.endReason === 'timeout'
+        return isDeadClick(frame as SlowClickFrame)
```

Comment:
Similar to below, all `ui.slowClickDetected` crumbs will always eval to "isDead == true" and a subset of that will be `isDeadRage == true`?

Reply:
yes

and same as below, i thinking about making sure the render code is in place, in case these things appear somehow. but the replayReader class is filtering them out right now